app/controllers/drinks_controller.rb

```diff
@@ -1,3 +1,17 @@
 class DrinksController < ApplicationController
-    
+    def index
+        if !params[:user_id]
```

Comment:
Devise is special in that it loads variables about users. So those variables `user_signed_in` and `current_user` that are passed to the react app on `index.html.erb` are also available globally through the rails app. It would also be possible to grab the user id off `current_user` in the controller instead of having to pass it from the front end through query params.